core/schema/secret.graphqls

```diff
@@ -1,6 +1,22 @@
 extend type Query {
   "Loads a secret from its ID."
   secret(id: SecretID!): Secret!
+
+  """
+  Sets a secret given a user defined name to its plaintext and returns the secret.
+  """
+  setSecret(
+
+  """
+  The user defined name for this secret
+  """
+  name: String!,
+
+  """
+  The plaintext of the secret
+  """
+  plaintext: String!
```

Comment:
uber nit: mostly whitespace, and it looks like we're trending away from commas

```suggestion
    """
    The user defined name for this secret
    """
    name: String!

    """
    The plaintext of the secret
    """
    plaintext: String!
```

Reply:
I like being coherent. :+1: